net-dialup/cutecom/cutecom-0.60.0_rc1-r1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake desktop xdg
+
+MY_PV=${PV/_rc/-RC}
+
+DESCRIPTION="A serial terminal, like minicom, written in Qt"
+HOMEPAGE="https://gitlab.com/cutecom/cutecom"
+SRC_URI="https://gitlab.com/cutecom/cutecom/-/archive/v${MY_PV}/cutecom-v${MY_PV}.tar.bz2"
+S="${WORKDIR}/cutecom-v${MY_PV}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="amd64 ~arm64 ~x86"
+
+DEPEND="
+	dev-qt/qtbase:6
```

Comment:
This is missing some major USE deps on `dev-qt/qtbase`, @benkohler:

From CMakeLists.txt:
```
find_package(Qt6Widgets REQUIRED)
find_package(Qt6Core REQUIRED)
find_package(Qt6Gui REQUIRED)
find_package(Qt6SerialPort REQUIRED)
find_package(Qt6Network REQUIRED)
```